photutils/psf/psfphotometry.py

```diff
@@ -0,0 +1,282 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Module which provides classes to perform PSF Photometry"""
+
+from __future__ import division
+import abc, six
```

Comment:
@bsipocz : You always know these things: Should this be an import of "six" (which requires users to have six installed) or "astropy.extern.six" (because users certainly have astropy installed anyway)?